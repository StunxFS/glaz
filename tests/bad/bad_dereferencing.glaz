fn main() {
  let x = 10;
  unsafe { _ = x.*; }
}
