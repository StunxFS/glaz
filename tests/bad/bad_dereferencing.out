tests/bad/bad_dereferencing.glaz:3:17: error: invalid indirect for `i32`
     | 
   3 |   unsafe { _ = x.*; }
     |                ^^^
glaz error: could not compile `bad_dereferencing`, aborting due to previous error.
